src/analysis/inner_product_space/pi_L2.lean

```diff
@@ -220,6 +222,16 @@ by { conv_rhs { rw ← complex.isometry_euclidean_proj_eq_self z }, simp }
   complex.isometry_euclidean z 1 = z.im :=
 by { conv_rhs { rw ← complex.isometry_euclidean_proj_eq_self z }, simp }
 
+/-- The isometry between `ℂ` and a two-dimensional real inner product space given by a basis. -/
+def complex.isometry_of_orthonormal {v : basis (fin 2) ℝ F} (hv : orthonormal ℝ v) : ℂ ≃ₗᵢ[ℝ] F :=
+complex.isometry_euclidean.trans (v.isometry_euclidean_of_orthonormal hv).symm
```

Comment:
The lemma I have in this PR simply reflects what I've needed so far for oriented angles. If I find I need lemmas about `(complex.isometry_of_orthonormal hv).symm` (e.g. explicit statements of what it does to basis vectors) I expect I'll add them when I need them.

Reply:
I think it would be good to add some lemmas characterising the new definition.

For example:
```lean
@[simp] lemma complex.map_isometry_of_orthonormal_symm_apply
  {v : basis (fin 2) ℝ F} (hv : orthonormal ℝ v) (f : F) :
  (complex.isometry_of_orthonormal hv).symm f = (v.coord 0 f : ℂ) + (v.coord 1 f : ℂ) * I :=
sorry

@[simp] lemma complex.map_isometry_of_orthonormal_apply
  {v : basis (fin 2) ℝ F} (hv : orthonormal ℝ v) (z : ℂ) :
  complex.isometry_of_orthonormal hv z = z.re • v 0 + z.im • v 1 :=
sorry
```

Reply:
I've added such lemmas. I didn't mark them as `simp` lemmas since I don't think splitting up by individual basis vectors is necessarily a simplification (certainly it's not of use for anything I've done with `complex.isometry_of_orthonormal` so far in working on oriented angles, though maybe there will be a one-off use of it when it comes to relating oriented and unoriented angles).